stl/inc/ranges

```diff
@@ -28,6 +28,10 @@ namespace ranges {
     // CONCEPT ranges::common_range
     template <class _Rng>
     concept common_range = range<_Rng> && same_as<iterator_t<_Rng>, sentinel_t<_Rng>>;
+
+    // CONCEPT ranges::viewable_range
+    template <class _Rng>
+    concept viewable_range = range<_Rng> && (safe_range<_Rng> || view<remove_cvref_t<_Rng>>);

```

Comment:
WG21-N4842 depicts `decay_t`. Is it okay to replace that with `remove_cvref_t`? (If so, should there be an LWG issue to simplify the Standardese accordingly?)

Reply:
Never mind, I see that Casey requested this, and submitted an editorial issue, addressing this feedback before I wrote it. Nice!

Reply:
See https://github.com/microsoft/STL/pull/389#discussion_r358447982.